lib/utils/Database.cpp

```diff
@@ -22,4 +31,18 @@ Database::~Database()
 {
 }
 
+bool
+Database::create(const std::string& file_name)
+{
+    std::cout << "Creating file: " << file_name << std::endl;
```

Comment:
For these `cout` calls, should we store the MPI rank in the class and do `cout` only for rank 0?

Reply:
This would make sense for mpi-I/O, where multiple processors read/write a single file. Current implementation is file-per-processor, and Database uses a separate file name per processor. And each filename is printed out in the current implemeation.

This convention could be changed, which I think we can discuss aside from this PR. I'm actually in need of MPI-I/O, and planning to write an database class for it. Maybe we can update our data formats/print-out messages in the next PR.